app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+import json
+
+URL = "https://api.weatherapi.com/v1/current.json?"
+FILTERING = "Paris"
+KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    result_from_weather_api = requests.get(URL + f"q={FILTERING}&key={KEY}")
```

Comment:
You can use params instead of the string concatenation
```suggestion
    result_from_weather_api = requests.get(URL, params={"q": FILTERING, "key": KEY)
```